### ISSUE TITLE
Training (fine-tuning) code

### ISSUE THREAD

**Issue by greeneggsandyaml** [FreddeFrallan/Multilingual-CLIP#1]
Hi authors, 

This repo looks amazing! I'm very excited to start using it. Are you planning on releasing the training/fine-tuning code?

Best,

**Comment by FreddeFrallan** [comment]
Hey!

Thank you for the kind words, hope you find the models useful. We do plan on releasing both the code and some pre-computed and translated training data, which would significantly speed up the training process for new models.
Hopefully I'll get around to organizing that by the end of the week! :)

Cheers

**Comment by greeneggsandyaml** [comment]
Wonderful, I'm excited to for the release of the training code and data!

**Comment by FreddeFrallan** [comment]
Hello again!

I just pushed up the code which was used during training (Tensorflow 2). It ain't pretty, and it will require you to prepare your own parallel corpus. Given time, I can perhaps also organise the code we used for translating and generating CLIP embeddings.
However, I also uploaded our already computed data in this[ Drive Folder.](https://drive.google.com/drive/folders/1I9a7naSZubUATWzLFv61DQMWyFlF7wR5)

Hope this at least gives you a good starting place for training your own models!
